ci/pipelines/deploy.yml

```diff
@@ -304,15 +304,22 @@ jobs:
     serial_groups: [directdebit-frontend-stg]
     plan:
       - get: omnibus
-      - get: directdebit-frontend
+      - get: git-directdebit-frontend-pre-release
         trigger: true
+      - task: download-directdebit-frontend-artifact
+        file: omnibus/ci/tasks/download-deployment-artifact.yml
+        input_mapping: { git-release: git-directdebit-frontend-pre-release }
+        output_mapping: { artifact: directdebit-frontend-artifact }
+      - put: paas-staging
         params:
```

Comment:
Maybe make this a YAML anchor and then you can use generic `path: artifact` and `manifest: artifact/manifest.yml` (don't think you even need the latter as it's the default).

Reply:
Oh ok, I thought that the inputs and outputs were name spaced at the plan level so we'd need unique names for every archive per job (hence the mapping). Otherwise yes, using generic names will allow us to use an anchor and simplify it all. 

The `manifest` may actually be a hangover from the `cf` resource I was originally using where it's mandatory. I can try removing it when using the `cf-cli`.